intel-retail/automated-self-checkout
# feat: Add performance mode parameter

## PR Checklist



## What are you changing?
Add performance mode parameter

## Issue this PR will close

close: #55

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
run benchmark.sh using
- '--performance_mode performance'
- '--performance_mode powersave'
- no '--performance_mode' set (should default to powersave

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
